README.md

```diff
@@ -22,7 +22,7 @@ Here are some resources and information about VictoriaMetrics:
 
 - Documentation: [docs.victoriametrics.com](https://docs.victoriametrics.com)
 - Case studies: [Grammarly, Roblox, Wix,...](https://docs.victoriametrics.com/casestudies/).
-- Available: [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/latest), [Docker images](https://hub.docker.com/r/victoriametrics/victoria-metrics/), [Source code](https://github.com/VictoriaMetrics/VictoriaMetrics)
+- Available: [Binary releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/latest), docker images [Docker Hub](https://hub.docker.com/r/victoriametrics/victoria-metrics/) and [Quay](https://quay.io/repository/victoriametrics/victoria-metrics), [Source code](https://github.com/VictoriaMetrics/VictoriaMetrics)
```

Comment:
Looks like we need to actualize and port repo descriptions from docker hub to quay. Otherwise, it shows blank pages.
If you'd like @zekker6 , I can do it later.

Reply:
I can do that as well. Didn't go ahead with this on Quay as readme pages are also missing on Docker Hub and I wasn't sure if that's supposed to be like that or not :)

Reply:
Thanks! I've updated and added some docs to both: docker hub and quay.